android/app/src/main/java/app/organicmaps/search/TabAdapter.java

```diff
@@ -72,8 +75,9 @@ private class PageChangedListener extends TabLayout.TabLayoutOnPageChangeListene
     public void onPageSelected(int position)
     {
       super.onPageSelected(position);
-      if (mTabSelectedListener != null)
+      if (mTabSelectedListener != null) {
         mTabSelectedListener.onTabSelected(Tab.values()[position]);
+      }
```

Comment:
Please follow the repo's code style.
```suggestion
      if (mTabSelectedListener != null)
        mTabSelectedListener.onTabSelected(Tab.values()[position]);
```

Reply:
I will make the necessary changes and provide an update, thanks!